void main(){

  print("welcome");

try{
  int div = 10 ~/ 0;
  print(div);
}
catch(obj){
  print("Exception occured $obj");
}
  print("hello");

}